### PR TITLE
doc: demos: Function Plotter stylistic improvements

### PR DESCRIPTION
**Pull request by markusgritsch**
Refactored code to be more DRY and more generic.

### CODE REVIEW

**Comment by fscherf** [comment]
Nice! Has the clip for the doc page to be redone?
Can you run isort to make the linter happy again (you can use `make isort` if you are on unix)

**Comment by markusgritsch** [comment]
Giving up on making isort happy. This is what isort thinks the imports should look like when running on my computer using
```
>>> import isort
>>> isort.file('plotter_picocss.py')
```
Cannot run `make isort` since I'm on Windows.  Furthermore I think the semantic position of the additional `NumberInput` in the original `input` statement was totally fine.

**Comment by fscherf** [comment]
@markusgritsch: No problem, thanks for trying. I ran `make isort` on your branch and pushed the resulting patch. If you are ok with this change, please squash all commits together

File: doc/content/demos/function-plotter/function-plotter.py [hunk 1]
```diff
@@ -77,20 +63,14 @@ def handle_request(self, request):
...
-            'sin(x)',
-            handle_change=self.handle_text_input_change,
-        )
-
-        self.f3 = TextInput(
-            'e**(-x*x)',
-            handle_change=self.handle_text_input_change,
-        )
+        self.f_list = []
+        for f in ['x*x/30-0.5', 'sin(x)', 'e**(-x*x)', 'sin(x)/x']:
```
Comment (fscherf) [inline doc/content/demos/function-plotter/function-plotter.py:67]: It seems the functions change. Do i have to rerecord the clip?

**Review by fscherf — CHANGES_REQUESTED** [review]
Could you please clean up the commit message? The first two lines will be enough. Also it has my sign-off in it now

**Comment by fscherf** [comment]
@markusgritsch: I think that's close enough for now :) thanks for updating!

**Comment by fscherf** [comment]
@markusgritsch: The docs are deployed to lona-web.org

**Comment by markusgritsch** [comment]
schwere Geburt :)

**Pull request merged**